src/components/__tests__/loginType.test.js

```diff
@@ -0,0 +1,24 @@
+// react libraries
+import React from 'react';
+
+// enzyme third party libraries
+import Enzyme, { shallow } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+
+// component
+import LoginType from '../login/LoginType';
+
+Enzyme.configure({ adapter: new Adapter() });
```

Comment:
#### Critical
Refactor this i.e. the `enzyme` setup should go into a file e.g. `testSetup` file then add the key to your `jest` configuration in `package.json`.

Also, do this for all the test files.

For more clarity check [Using `enzyme` with React16](https://airbnb.io/enzyme/docs/installation/react-16.html) and [`enzyme` setup](https://stackoverflow.com/questions/46627353/where-should-the-enzyme-setup-file-be-written)